### DISCUSSION TITLE
FiniteStrainCrystalPlasticity: Constitutive failure

### DISCUSSION THREAD

**Discussion by engrmessi** [idaholab/moose#26248] [Q&A Modules: Solid mechanics]
## Bug Description
the following error occurred in the object "crysp", of type "finitestraincpsliprateres". finitestrain crystal plasticity: constitutive failure?[39m

## Impact
The solution did not converge. Please i need help to fix this problem. Thanks

**Comment by jiangwen84** [comment]
You need to be more specific on what causes failure.  You need to check if your simulation condition is reasonable.  I suggest you ask your question under Discussion not the Issue here.

**Reply by lindsayad** [reply]
Migrated
